### ISSUE TITLE
Cannot Download attachments after domain change

### ISSUE THREAD

**Issue by SecureCPU** [freescout-helpdesk/freescout#2487]
Hey,
I'm running FreeScout on Ubuntu.
Recently I just changed the domain for our Freescount instance.
However, since the change, We cannot download any past attachments for files.

I changed the domain by editing the .env file then running:
`sudo php artisan freescout:clear-cache`
`sudo chown -R www-data:www-data /var/www/html`

**Comment by SecureCPU** [comment]
So for some reason, its not just the attachments sent before the domain change. 
Even the files sent today, can't be downloaded. I keep getting `Failed - Forbidden `

I can't access any files sent at all.

Can someone please help me?

**Comment by SecureCPU** [comment]
Would this have anything do to with extended attachments module?

**Comment by freescout-helpdesk** [comment]
There is no connection between attachments and FreeScout URL.

Make sure your have `location ~* ^/storage/attachment/` in nginx: https://github.com/freescout-helpdesk/freescout/wiki/Installation-Guide#61-nginx

Make sure you have not changed `APP_KEY` in .env file.

**Comment by SecureCPU** [comment]
I just checked and the settings are correct.
and i didn't change the APP_KEY

**Comment by SecureCPU** [comment]
This is the error when i sent myself a pdf attachment:

![Screenshot_20221222_100544](https://user-images.githubusercontent.com/99109793/209163209-6bc9c401-7f2e-40ec-90bb-99fdc0f6f8fc.png)

**Comment by SecureCPU** [comment]
I found the issue.
The Issue wasn't with Freescout. It was with Cloudflare.

Under Cashing > Configuration > Cashing Level
It has to be set to `Standard`

![Screenshot_20221222_104524](https://user-images.githubusercontent.com/99109793/209171348-b6ad37ef-0d48-47fd-bdbc-fa04845b5cb3.png)

Previously it was set to `Ignore query string` Once I changed it to `Standard` I was able to view attachments again
